docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -205,3 +205,49 @@ Le code ci-dessus génère ce qui suit sur la page :
 Une liste de toutes les icônes disponibles est affichée ci-dessous avec leurs noms associés. Cliquez sur une icône pour copier le code du composant.
 
 <IconsList />
+
+### Code
+
+Utilisez le composant `<Code>` pour rendre le code en évidence syntaxique lorsque l'utilisation d'un [bloc de code Markdown](/fr/guides/authoring-content/#blocs-de-code) n'est pas possible, par exemple, pour rendre des données provenant de sources externes comme des fichiers, des bases de données ou des API.
+
+Voir la documentation [Expressive Code "Code Component"](https://expressive-code.com/key-features/code-component/) pour plus de détails sur les fonctions que `<Code>` supporte.
+
+```mdx
+# src/content/docs/example.mdx
+
+import { Code } from '@astrojs/starlight/components';
+
+export const exampleCode = `console.log('Cela peut provenir d'un fichier ou d'un CMS !');`;
+export const fileName = 'example.js';
+export const highlights = ["file", "CMS"];
+
+<Code code={exampleCode} lang="js" title={fileName} mark={highlights} />
+```
+
+Le code ci-dessus génère la page suivante :
```

Comment:
```suggestion
Le code ci-dessus génère ce qui suit sur la page :
```

This technically does not generate a page so I changed to something closer to the original text.